installer/manifests/keptn/values.yaml

```diff
@@ -59,6 +59,7 @@ features:
   oauth:
     enabled: false
     prefix: "keptn:"
+  gitRemoteURLDenyList: ""
```

Comment:
should we rather add a `git` object here to group together other potential git related config values?

Reply:
applied, thx